src/app/issues-viewer/card-view/card-view.component.css

```diff
@@ -133,3 +133,15 @@ div.column-header .mat-card-header {
   display: inline-flex;
   font-size: 14px;
 }
+
+/deep/ .pagination-hide-arrow .mat-paginator-navigation-previous {
+  display: none !important;
+}
+
+/deep/ .pagination-hide-arrow .mat-paginator-navigation-next {
+  display: none !important;
+}
+
+/deep/ .mat-paginator-range-actions {
+  height: 47px;
+}
```

Comment:
/deep/ is deprecated, please replace with :host ::ng-deep in line with https://angular.io/guide/component-styles

Reply:
While this may seem hacky, I could not find a proper solution and there is no appropriate API for mat-pagination which I am sure you looked at as well.
Good job looking through the inspector and finding this workaround! 👍

Reply:
Fixed!